yarn.lock

```diff
@@ -1909,7 +1909,7 @@ eslint-plugin-react@7.11.1:
     prop-types "^15.6.2"
 
 "eslint-plugin-shopify@file:./.":
-  version "25.1.0"
+  version "26.0.0"
```

Comment:
This repo lints itself using itself. When we release a new version, this file gets updated with the new version of itself. We probably forgot to do this in the last release.

Reply:
🔁